src/test/java/gameframework/motion/blocking/MoveBlockerRulesApplierDefaultImplTest.java

```diff
@@ -1,5 +1,8 @@
 package gameframework.motion.blocking;
 
+import static org.junit.Assert.assertEquals;
```

Comment:
again, please avoid touching lines you don't change. Keep the diff as simple as possible to facilitate code review.